src/pubtools/_quay/operator_pusher.py

```diff
@@ -388,6 +388,58 @@ def iib_remove_operators(
             env_vars,
         )
 
+    @classmethod
+    @tx.instrument_func()
+    def iib_add_deprecations(
+        cls,
+        index_image: Optional[str] = None,
```

Comment:
for add_deprecations index_image should not be optional

Reply:
Thanks. Removed default of it.